app/components/autocomplete/at_mention_item/at_mention_item.tsx

```diff
@@ -50,18 +52,19 @@ const getStyleFromTheme = makeStyleSheetFromTheme((theme: Theme) => {
         },
         rowInfo: {
             flexDirection: 'row',
-            flex: 1,
+            width: '100%',
         },
         rowFullname: {
             fontSize: 15,
             color: theme.centerChannelColor,
             paddingLeft: 4,
+            maxWidth: '42%',
         },
         rowUsername: {
             color: theme.centerChannelColor,
             fontSize: 15,
             opacity: 0.56,
-            flex: 1,
+            maxWidth: '45%',
         },
```

Comment:
Should we only make these CSS changes in rows where we need to show the custom status?